code/modules/vehicles/van/box_van.dm

```diff
@@ -0,0 +1,290 @@
+
+//Trucks
+//Read the documentation in multitile.dm before trying to decipher this stuff
+
+/obj/vehicle/multitile/box_van
+	name = "Colony Van"
+	desc = "A rather old hunk of metal with four wheels, you know what to do. Entrance on the back and sides."
+	layer = ABOVE_XENO_LAYER
+
+	icon = 'icons/obj/vehicles/box_van.dmi'
+	icon_state = "van_base"
+
+	bound_width = 64
+	bound_height = 64
+
+	bound_x = 0
+	bound_y = 0
+
+	interior_map = /datum/map_template/interior/box_van
+
+	entrances = list(
+		"left" = list(2, 0),
+		"right" = list(-1, 0),
+		"back_left" = list(1, 2),
+		"back_right" = list(0, 2)
+	)
+
+	vehicle_flags = VEHICLE_CLASS_WEAK
+
+	passengers_slots = 8
+	xenos_slots = 2
+
+	misc_multipliers = list(
+		"move" = 0.5, // fucking annoying how this is the only way to modify speed
+		"accuracy" = 1,
+		"cooldown" = 1
+	)
+
+	movement_sound = 'sound/vehicles/tank_driving.ogg'
+	honk_sound = 'sound/vehicles/honk_2_truck.ogg'
+
+	vehicle_light_range = 8
+
+	move_max_momentum = 3
+
+	hardpoints_allowed = list(
+		/obj/item/hardpoint/locomotion/van_wheels,
+	)
+
+	move_turn_momentum_loss_factor = 1
+
+	req_access = list()
+	req_one_access = list()
+
+	door_locked = FALSE
+
+	mob_size_required_to_hit = MOB_SIZE_XENO
+
+	var/overdrive_next = 0
+	var/overdrive_cooldown = 15 SECONDS
+	var/overdrive_duration = 3 SECONDS
+	var/overdrive_speed_mult = 0.3 // Additive (30% more speed, adds to 80% more speed)
+
+	var/momentum_loss_on_weeds_factor = 0.2
+
+	move_on_turn = TRUE
+
+	var/list/mobs_under = list()
+	var/image/under_image
+	var/image/normal_image
+
+	var/next_push = 0
+	var/push_delay = 0.5 SECONDS
+
+/obj/vehicle/multitile/box_van/Initialize()
+	. = ..()
+	under_image = image(icon, src, icon_state, layer = BELOW_MOB_LAYER)
+	under_image.alpha = 127
+
+	normal_image = image(icon, src, icon_state, layer = layer)
+
+	icon_state = null
+
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_LOGGED_IN, PROC_REF(add_default_image))
+
+	for(var/I in GLOB.player_list)
```

Comment:
Single letter variables. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations

Reply:
Not even sure what "I" is supposed to be an abbreviations of? Initialize? Aaaa..

Reply:
for(var/player in GLOB.player_list)